locale/en.yml

```diff
@@ -886,6 +887,7 @@ en:
       OrchestrationTemplateVnfd: VNF Template
       ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate: vApp Template
       PersistentVolume:         Persistent Volume
+      PhysicalServer:           Physical Server
```

Comment:
@AparnaKarve Yes, it is needed for us being able to translate breadcrumb in physical server list view.